code_annotations/base.py

```diff
@@ -314,7 +315,11 @@ def __init__(self, config):
         """
         self.config = config
         self.echo = self.config.echo
+        # errors contains formatted error messages
         self.errors = []
+        # annotation_errors contains (annotation, AnnotationErrorType, args) tuples
+        # This attribute may be parsed by 3rd-parties, such as edx-lint.
+        self.annotation_errors = []
```

Comment:
If this is required to keep edx-lint happy, is it possible to unit test to ensure it keeps working?

Reply:
:ok_hand: